contracts/price_oracle/price_feed/PtPriceFeed.sol

```diff
@@ -60,7 +64,14 @@ contract PtPriceFeed is IPriceFeed {
     /// @param pendleMarket_ Address of the Pendle market
     /// @param twapWindow_ Duration of TWAP window (recommended 15 minutes)
     /// @param priceMiddleware_ Address of price oracle middleware that must support the underlying asset
-    constructor(address pendleOracle_, address pendleMarket_, uint32 twapWindow_, address priceMiddleware_) {
+    /// @param usePendleOracleMethod 0 for getPtToSyRate, 1 for getPtToAssetRate
+    constructor(
+        address pendleOracle_,
+        address pendleMarket_,
+        uint32 twapWindow_,
+        address priceMiddleware_,
+        uint256 usePendleOracleMethod
```

Comment:
Why this parameter  is here usePendleOracleMethod - is not used inside constructor

Reply:
I had uncommitted changes. 🫣